src/_lib/sass/src/_popovers.scss

```diff
@@ -14,119 +15,166 @@
     // So reset our font and text properties to avoid inheriting weird values.
     @include reset-text;
     font-size: $font-size-base;
-  
+
     color: get-color("gray", base);
     background-color: $popover-bg;
     background-clip: padding-box;
-    // border: 1px solid $popover-fallback-border-color;
-    // border: 1px solid $popover-border-color;
-    // border-radius: $border-radius-large;
-    // @include box-shadow(0 5px 10px rgba(0,0,0,.2));
-  
+    border: 1px solid $popover-border-color;
+    border-radius: 3px;
+    box-shadow: 0 0 8px rgba(0,0,0,.2);
+    font-family: 'Proxima Nova Regular', Arial, sans-serif;
+    z-index: 999;
```

Comment:
Is there a reason for z-index 999? We have a master list of z-index set in the variables file, with popovers currently at `1060`. This is for controlling how certain elements appear in relation to each, e.g popovers within modals, toasts over modals etc

Reply:
We had some issues in analytics with popover appearing above the dash dropdown.

Reply:
Removed - I can deal with it in analytics.